resources/welcome/index.html

```diff
@@ -169,38 +187,367 @@ <h3 id="generate-app">Generate Azure IoT application</h3>
+                    <p>You can use <a href="https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-toolkit">Azure IoT Hub Toolkit</a> (formerly Azure IoT Toolkit) to monitor device-to-cloud (D2C) messages in IoT Hub.</p>
+                    <ol>
+                        <li>
+                            <p>In Visual Studio Code, look for <strong>Azure IoT Hub Toolkit</strong> in the extension marketplace and install it if you have not done that.</p>
+                        </li>
+                        <li>
+                            <p>Expand the <strong>AZURE IOT HUB DEVICES</strong> pane on the left, click <strong>Select IoT Hub</strong>. It will guide you to sign in to Azure if you have not yet done that.</p>
+                        </li>
+                        <li>
+                            <p>After signing in, select an Azure subscription and then Azure IoT Hub to use. The device list for that Azure IoT Hub will display in the pane.
+                                <img src="https://raw.githubusercontent.com/Microsoft/vscode-azure-iot-tools/master/resources/welcome/device-workbench-images/select-iothub.gif" alt="Select IoT Hub" /></p>
+                        </li>
+                        <li>
+                            <p>In the <strong>AZURE IOT HUB DEVICES</strong> pane right click on the device name you created and select
+                                <strong>Start Monitoring D2C Message</strong>.
+                                <img src="https://raw.githubusercontent.com/Microsoft/vscode-azure-iot-tools/master/resources/welcome/device-workbench-images/monitor-d2c.png" alt="Monitor D2C Message" /></p>
+                        </li>
+                        <li>
+                            <p>In <strong>OUTPUT</strong> pane, you can see the incoming D2C messages to the IoT Hub.
+                                <img src="https://raw.githubusercontent.com/Microsoft/vscode-azure-iot-tools/master/resources/welcome/device-workbench-images/d2c-output.png" alt="D2C message" /></p>
+                        </li>
+                    </ol>
+                    <h3 id="problemsandfeedback">Problems and feedback</h3>
+                    <p>If you encounter problems, you can check for a solution in the <a href="https://microsoft.github.io/azure-iot-developer-kit/docs/faq/">IoT DevKit FAQ</a> or reach out to us from <a href="https://gitter.im/Microsoft/azure-iot-developer-kit">Gitter</a>.
+                        You can also give us feedback by leaving a comment on this page.</p>
+                    <h3 id="learnmore">Learn more</h3>
+                    <p>To learn more about using Azure IoT Device Workbench, visit the Github repo: <a href="https://github.com/Microsoft/vscode-iot-workbench">https://github.com/Microsoft/vscode-iot-workbench</a>.</p>
                 </div>
-            </footer>
+                <input type="checkbox" id="never-show" {{Checked}}>Never show welcome page again<br>
```

Comment:
Better naming, align with vscode and other extensions, by default the checkbox is checked.
VS Code welcome page
![image](https://user-images.githubusercontent.com/1050213/51510703-20933c00-1e39-11e9-9ad7-e0bad093627b.png)
GitLens:
![image](https://user-images.githubusercontent.com/1050213/51510728-3acd1a00-1e39-11e9-998a-491eddfaaf41.png)

Reply:
Use "Show welcome page after upgrading to new version" instead